wsclient/src/test/java/org/hpccsystems/ws/client/platform/DFUFileDetailInfoTest.java

```diff
@@ -35,6 +43,9 @@ public static DFUDataColumnInfo getColumnByName(final DFURecordDefInfo parent, f
     @Test
     public void testGetRecordEcl() throws Exception {
         EclRecordInfo info = DFUFileDetailInfo.getRecordEcl(WITH_ANNOTATION);
+        if (info.getParseErrors().size()>0) {
```

Comment:
if size is guaranteed to be >= 0, it could be more efficient to just test for .size()==0

Reply:
I don't understand your meaning, can you give an example?  if the size >0 I want it to fail.

Reply:
I guess the comment doesn't apply to java (since there's no unsigned int)...
Basically if you're guranteed that size() would always return 0 or greater, your conditional could have been if (info.getParseErrors().size()!=0), which is slightly more efficient than if (info.getParseErrors().size()>0)